### PR TITLE
[GIZ] Add LiqPay payment processor

### PR DESCRIPTION
**Pull request by arsentur**
**Description:**
- Cherry-pick commits which add LiqPay payment processor into e-commerce
- Move version from python 2 to 3 

**Youtrack:**
[GIZ-309](https://youtrack.raccoongang.com/issue/GIZ-309)

**DevOps info:**
Please change next values for the ecommerce.yml under `JWT_AUTH`

```ecommerce.yml
JWT_ISSUERS:
    -   AUDIENCE: {audience}
        ISSUER: https://lms-giz-dev.raccoongang.com/oauth2
        SECRET_KEY: {secret}
```

### CODE REVIEW

File: ecommerce/extensions/payment/views/liqpay.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
+""" Views for interacting with the LiqPay payment processor. """
+from __future__ import unicode_literals
```
Comment (Golub-Sergey) [inline ecommerce/extensions/payment/views/liqpay.py]: this line can be removed

File: ecommerce/extensions/payment/processors/liqpay.py [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+        super(Liqpay, self).__init__(site)
+        # NOTE(smandziuk):
+        # - To enable Liqpay sandbox mode add to config entry - 'sandbox_mode': 1,
+        # - To change version of liqpay API add to config entry - 'version' (on october 2019 api version = '3')
+        self.sandbox_mode = self.configuration.get('sandbox_mode', 0)
+        self.version = self.configuration.get('version', '3')
+
+    @property
+    def cancel_url(self):
+        return get_ecommerce_url(self.configuration['cancel_checkout_path'])
```
Comment (Golub-Sergey) [inline ecommerce/extensions/payment/processors/liqpay.py:46]: `cancel_checkout_path` is always exists in the `self.configuration`?

**Review by Golub-Sergey — APPROVED** [review]
this comments are optional to resolve

File: ecommerce/extensions/payment/processors/liqpay.py (hunk 2 above)
Comment (vladhrech) [reply ecommerce/extensions/payment/processors/liqpay.py:46]: Yes, it's always exists

**Pull request merged**